src/components/App/App.js

```diff
@@ -165,8 +165,14 @@ class App extends Component {
 
   renderAllAnnotations() {
     const as = this.props.store.annotationStore;
+    const entities = [...as.annotations, ...as.predictions];
 
-    return <Grid store={as} annotations={[...as.annotations, ...as.predictions]} root={as.root} />;
+    if (isFF(FF_SIMPLE_INIT)) {
+      // the same sorting we have in AnnotationsCarousel, so we'll see the same order in both places
+      entities.sort((a, b) => new Date(b.createdDate).getTime() - new Date(a.createdDate).getTime());
```

Comment:
if we have the same sorting function being used, maybe a good idea to define it some where in `lsf/src/utils/date.js` or `utilities.js` and reuse it in both spots from there - chance are if the code needs to change down the line we'll need to update both locations at the same time